scripts/run.js

```diff
@@ -6,22 +6,30 @@ const { exec, execSync } = require("child_process");
 const PORT = 5050;
 const systemType = os.platform();
 
-if (systemType !== "darwin") {
-  console.log("This command currently only supports unix at the moment.");
-  return;
-}
-
 function newTerminal(command) {
-  const PWD = path.join(__dirname, "../");
-  execSync(`
-  osascript -e 'tell application "Terminal" to activate' \
-  -e 'tell application "System Events" to keystroke "t" using {command down}' \
-  -e 'tell application "Terminal" to do script "cd ${PWD} && ${command}" in front window'
-  `);
+  const CWD = path.join(__dirname, "../");
+
+  // empty double quotes is intentional because of how the "start" Windows
+  // command parameters work
+  if (systemType === "win32") {
+    setTimeout(() => {
+      exec(`start "" /d ${CWD} ${command}`);
+    }, 1000);
+  } else if (systemType === "darwin") {
+    execSync(`
+      osascript -e 'tell application "Terminal" to activate' \
+      -e 'tell application "System Events" to keystroke "t" using {command down}' \
+      -e 'tell application "Terminal" to do script "cd ${CWD} && ${command}" in front window'
+      `);
+  } else
+    console.log(
+      "This dev script currently only supports Windows/unix at the moment."
+    );
 }
 
 // Make sure all instances of ngrok's process doesn't exist.
-exec("killall ngrok");
+if (systemType === "win32") exec('Taskkill /IM "ngrok.exe" /F');
+else if (systemType === "darwin") exec("killall ngrok");
```

Comment:
Please have curly braces for all statements, even if it's not required.
Also, please have the next statement on the next line.

This will allow better readability and keeps the codebase consistent.

Reply:
resolved on commit [bb7b3e2](https://github.com/cppsea/icebreak/pull/3/commits/bb7b3e2ed0c5fec240287e223167e51dc985351a)